### PR TITLE
Add over_select benchmarks

### PR DESCRIPTION
**Pull request by bmarkons**
https://community.rubybench.org/t/can-sequel-be-configured-so-it-defer-materializes/159/9

### CODE REVIEW

File: pg/benchmarks/bm_scope_all_over_select.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Benchmark.pg("pg/scope_all_over_select", time: 5) do
+  str = ""
+
+  result = connection.exec("SELECT * FROM users")
+  result.type_map = type_mapper
+  
+  result.each_row do |row| 
+    str << "name: #{row[0]} email: #{row[1]} approved: #{row[2]} age: #{row[3]} birthday: #{row[4]} \n"
+  end
```
Comment (bmarkons) [inline pg/benchmarks/bm_scope_all_over_select.rb]: @SamSaffron will this do the work? Or you want to replace each blocks with while loop, even though we are using blocks in Sequel and Active Record benchmarks.

**Comment by SamSaffron** [comment]
Raw should always be the fastest possible implementation, measure on local
see what is fastest

**Comment by bmarkons** [comment]
Right, I'll switch to while loop then :racehorse:

**Comment by bmarkons** [comment]
@SamSaffron pls verify this for me before I merge. Thanks

**Comment by SamSaffron** [comment]
Looks good to me, let's get this backfilled

**Comment by bmarkons** [comment]
Sure, I'm on it :+1:

**Pull request merged**